tm/src/main/java/io/seata/tm/api/GlobalTransactionContext.java

```diff
@@ -34,7 +34,7 @@ private GlobalTransactionContext() {
      *
      * @return
      */
-    private static GlobalTransaction createNew() {
+    static GlobalTransaction createNew() {
```

Comment:
I think should include the
```
public
```
cause the code style is different in different people, we should keep it same.

Reply:
done